frontend/src/lib/components/SchemaForm.svelte

```diff
@@ -27,7 +27,7 @@
 				return
 			}
 
-			if (isCodeInjection(arg.value)) {
+			if (isCodeInjection(String(arg.value))) {
```

Comment:
do not think it's correct, you do not want to do rawValueString -> JSON.parse -> stringify
It's better to do instead
arg.rawValue ?? arg.value